files/demo.hledger

```diff
@@ -0,0 +1,178 @@
+﻿# Welcome to hledger, This file is where your financial history will be recorded.
+# It is a plain text file, so it is easy to edit and manipulate, and you can use
+# search and replace functionality in your text editor
+
+# Everything after '#', ';' or '*' are comments, so they are not read by hledger,
+# except in some cases to be discussed later
+
+
+# To start configuring our journal, it is a good idea to define a default currency
+D USD 1,000.00  ; Format - D Currency amountFormat
```

Comment:
".. and its number format" ?
Also let folks know that comma as decimal mark is supported, maybe show a commented alternative for them.

Reply:
Do you think I could just tell the user he could define different formats as in manual without explaining how here to avoid no essential feature?

Reply:
Sure, your call. Keeping it simple is good. If it were me, maybe I'd add a `# (or 1.000,00 or ` 000,00)` though since that's normal number format for probably half of users.